build.sbt

```diff
@@ -95,6 +95,25 @@ inThisBuild(
   )
 )
 
+onLoad.in(Global) ~= { old =>
+  import java.nio.file._
+  val sh =
+    if (scala.util.Properties.isWin) "#!C:/Program\\ Files/Git/usr/bin/sh.exe"
```

Comment:
Maybe use the condition to not write the hook on windows for now.

Reply:
Sounds good, done.